AirCastingTests/Utils/DummyError.swift

```diff
@@ -0,0 +1,13 @@
+// Created by Lunar on 19/06/2021.
+//
+
+import Foundation
+
+struct DummyError: Error, Equatable, LocalizedError {
+    let errorData: String
+    var errorDescription: String? { errorData }
+    
+    init(errorData: String = .default) {
+        self.errorData = errorData
+    }
```

Comment:
How about using `NSError(domain:"", code:0)` in test rather than defining this case. It would be more contextual in tests where the domain can be specyfic and give more context. Also `UUID().uuidString` would be more reliable to check if a static sting in development faze was not used.

Reply:
This class is used as a stub in testing scenarios that require stubbed dependencies to return an error and where the concrete error type, domain etc. don't matter at all. I think I made a mistake of being lazy and this `DummyError` should actually be completely empty implementation and for the tests where `localizedDescription` actually matter I shoul've introduced a `LocalizedErrorStub`. Will do so.